sig-multicluster/charter.md

```diff
@@ -0,0 +1,45 @@
+# SIG Multicluster Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The scope of SIG Multicluster includes:
+
+- Solving common challenges related to management of multiple Kubernetes clusters and the applications that exist therein
```

Comment:
This is really broad, could you scope it down?  

e.g. - what aspects?  
- Networking, application mgmt, disaster recovery, etc.

Reply:
I think the most sensible thing to say here is probably to be really specific about the scopes of the existing projects, which are:

- cluster-registry
- federation-v2
- kubemci

I don't think it's especially useful to have a vague prose description of the scope - since this SIG is essentially focused entirely around these projects at this point.

Reply:
> I think the most sensible thing to say here is probably to be really specific about the scopes of the existing projects 

Agreed, we want to make certain the areas of responsibility translate to specific areas of ownership so we start to rationalize the SIGs across the project.